### DISCUSSION TITLE
Welcome to ultraschall-portable Discussions!

### DISCUSSION THREAD

**Discussion by rstockm** [Ultraschall/ultraschall-portable#379] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hermanverbaeten** [comment]
Hi rsockm, my name is Herman and i'm from Belgium near Antwerp. I recently started using touchOSC Mk2 and now i'm trying to understand and figure out the commands in default.reaperOSC and logicpad.reaperOSC. And more specific i'm trying to find out what exactely the effect is of the commands (all in caps) in these files. In the beginning of the file a lot is explained and some commands are very obvious but some aren't, so i was wondering if you could tell me if there's a more detailed list of these commands with their meaning.  thanks for your advice!
Kind regards,
Herman
